src/border_agent/border_agent.cpp

```diff
@@ -278,9 +278,8 @@ void BorderAgent::PublishMeshCopService(void)
     txtList.emplace_back("dn", otThreadGetDomainName(instance));
 #endif
 
-    mPublisher->PublishService(/* aHostName */ nullptr, otBorderAgentGetUdpPort(instance),
-                               kBorderAgentServiceInstanceName, kBorderAgentServiceType, Mdns::Publisher::SubTypeList{},
-                               txtList);
+    mPublisher->PublishService(/* aHostName */ "", otBorderAgentGetUdpPort(instance), kBorderAgentServiceInstanceName,
```

Comment:
It seems this is not a doc change. Could you clarify the purpose of doing so? Or you may want to make this change in another PR.

Reply:
This isn't change in this PR, see https://github.com/openthread/ot-br-posix/pull/1073 instead.